lib/schemas/eikjson.schema.json

```diff
@@ -19,6 +19,12 @@
       "type": "string",
       "minLength": 1
     },
+    "type": {
+      "description": "The type of the Eik package. Must be one of 'package', 'npm' or 'map'",
```

Comment:
Wondering if this should help folks make a decision as to which one to choose. Would it be worth mentioning the matching url prefix? Does the different options need to be repeated from the enum below? Is this not included in the schema validation error?

Reply:
fixed